webrtc-stats.html

```diff
@@ -1749,6 +1750,21 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>voiceActivityFlag</code></dfn> of type <span class="idlMemberType"><a>
+                boolean</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for audio. Whether the last RTP packet sent or played out by this track
+                  contained voice activity or not based on the V bit of the extension header defined
+                  in [[RFC6464]], if present.
```

Comment:
consider rewording:

 based on the presence of the V bit in the extension header, as defined in [[RFC6464]]